### ISSUE TITLE
Как использовать в leaflet'e?

### ISSUE THREAD

**Issue by stdex** [KoGor/Maps.GeoInfo#1]
Здравствуйте. Не подскажите, как использовать в leaflet, а то там границы съезжают:
https://github.com/KoGor/Maps.GeoInfo/blob/master/russia_1e-7sr.json

Вообщем, похоже известная проблема в leaflet'e:
http://gis.stackexchange.com/questions/110424/stray-horizontal-lines-after-converting-from-geojson-to-topojson
https://github.com/mbostock/topojson/issues/242
http://stackoverflow.com/questions/21561960/leaflet-error-in-geojson-overlay-at-russia-finland-border
https://www.mapbox.com/mapbox.js/example/v1.0.0/omnivore-topojson/

**Comment by KoGor** [comment]
Да, разрезание по антимеридиану штука известная, ну в привиденных вами ссылках вполне исчерпывающая информация. Можете ещё про обратную задачу посмотреть для понимания: [How can I remove a line from the 110m TopoJson world map?](http://stackoverflow.com/questions/19041457/how-can-i-remove-a-line-from-the-110m-topojson-world-map)

Так что либо разрезать вручную, либо использовать GeoJSON, как-то так.
